src/commands/build.ts

```diff
@@ -285,99 +277,69 @@ export async function command(commandOptions: CommandOptions) {
             await fs.writeFile(cssOutPath, resources.css);
             code = `import './${path.basename(cssOutPath)}';\n` + code;
           }
+          const resolveImportSpecifier = createImportResolver({
+            fileLoc,
+            dependencyImportMap,
+            isBuild: true,
```

Comment:
@drwpow I see you using `isDev` in #514, and I think we use `isDev` in other places as well. I'll change to isDev for consistency